src/stories/Select.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import {Meta} from '@storybook/react'
+
+import {BaseStyles, Text, Select, ThemeProvider} from '..'
+import {SelectProps} from '../Select'
+
+export default {
+  title: 'Forms/Select',
+  component: Select,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
+  argTypes: {
+    block: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    contrast: {
```

Comment:
This shouldn't be relevant for this component

Reply:
Ok. I can remove it and we can always add it later if we decide we want parity with `TextInput`